tests/installation/reboot_after_installation.pm

```diff
@@ -35,8 +35,10 @@ sub run {
         my $svirt = console('svirt');
         $svirt->change_domain_element(os => boot => {dev => 'hd'});
     }
+    testapi::query_isotovideo('backend_set_ssh_half_open_expected');
```

Comment:
All calls should go over the testapi so I suggest without further functional impact to add corresponding functions to the testapi itself but first a conceptional thing: The test code should not care about if any console connection goes directly to the tty on a qemu VM or remotely over ssh. Why not what I suggested in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5778#pullrequestreview-164196772 ? At least: How about a method named `expect_console_disconnect`?

Reply:
for the api over testapi ,  because I'm  misunderstand your suggest way : define the function the the testcase , it doesn't work,  so i directly use it .
If backend code is accessible , I can move this part in to backend code.

Reply:
Yes, define a method in testapi.pm of os-autoinst, the backend. The code here could read like this

```suggestion
    expect_console_disconnect {
        send_key 'alt-o';    # Reboot
        power_action('reboot', observe => 1, keepconsole => 1, first_reboot => 1);
    }
```

and in testapi.pm:

```suggestion
sub expect_console_disconnect(&@) {
    my ($callback) = @_;
    # inform the backend/consoles that sockets will be closed from one side
    # to prevent "half-open socket" error
    query_isotovideo('backend_set_ssh_half_open_expected');
    $callback->() if $callback;
    query_isotovideo('backend_clean_ssh_half_open_expected');
}
```